### ISSUE TITLE
Sign Out > Opening the Sign in screen after sign out, it automatically signs in as a previous account

### ISSUE THREAD

**Issue by tlle14** [linagora/Twake-Mobile#899]
# Precondition
- Sign in as a UserA

# Steps and Actual Result
This issue randomly happens

1. Open My Account screen
2. Tap on "Sign Out" button
3. After Signed out, it redirects to the Welcome Screen
4. Tap on "Sign In" button on the Welcome screen
5. Verify issue: Instead of opening Sign In page, it continues sign in as UserA. 

# Expected Result
The user should be able to Sign out successfully.
It shouldn't re-login when opening Sign In screen

# Environment
- Build version: 3.0.5
- Device:
Samsung Galaxy A51 - Android 11 - 6.5 inch
- QA environment

**Comment by bobs4462** [comment]
Hello, how often does it happen? 
Might be internet connection issues, in order to sign out, the app needs to briefly open the browser and clear the cookies, if internet connection is unstable it might not work.
Also QA environment is pretty unstable lately, so again it might be backend issue. From mobile application we can only open the browser, send logout request and pray that it will work :)

**Comment by tlle14** [comment]
Hi @bobs4462 
This issue happens randomly. 
Can we do something on App to prevent this situation? 
Such as: Do not show the Welcome screen when the cookie on App is still active?
And as confirmed before, I understand that when logout on App, it is still logged in on Browser.

**Comment by bobs4462** [comment]
no, alas nothing can be done by the app, the browser are designed in such a way that you cannot clear it's cookies. Let's just wait for the QA to become a little stable and test again. I for one cannot reproduce this issue

**Comment by tlle14** [comment]
Okay thank @bobs4462 I will retest after the QA becomes stable.

**Comment by RomaricMourgues** [comment]
@RomaricMourgues for the record I can't do anything as this is related to the use of LemonLDAP. Replacement is planed in october.
